packages/design/src/Animation.stories.mdx

```diff
@@ -73,9 +73,3 @@ side nav, or accordion.
 ### Linear
 
 Use for color and opacity changes (things that don’t move).
-
```

Comment:
If you, dear reviewer, can find this token being defined somewhere in the codebase, by all means these docs can stay 😄